test/Tests/src/Data/Map_Spec.enso

```diff
@@ -445,7 +450,7 @@ spec =
             map.to_vector.should_equal [["A", 1], ["B", 2], ["C", 3]]
 
         Test.specify "should treat Java Map as Enso map" <|
-            sort_by_keys vec = vec.sort by=x-> y-> x.first.compare_to y.first
+            sort_by_keys vec = vec.sort by=x-> y-> Ordering.compare x.first y.first
```

Comment:
Oh, I see. `Ordering.compare` is a static method to compare two objects and get an `Ordering`. OK, that seems fine.